Dockerfile

```diff
@@ -17,12 +17,30 @@ RUN mkdir /opt/android-ndk-tmp
 RUN cd /opt/android-ndk-tmp && wget -q http://dl.google.com/android/repository/android-ndk-r12-linux-x86_64.zip
 # uncompress
 RUN cd /opt/android-ndk-tmp && unzip -q android-ndk-r12-linux-x86_64.zip
-# move to it's final location
+# move to its final location
 RUN cd /opt/android-ndk-tmp && mv ./android-ndk-r12 /opt/android-ndk
 # remove temp dir
 RUN rm -rf /opt/android-ndk-tmp
 # add to PATH
 ENV PATH ${PATH}:${ANDROID_NDK_HOME}
+
+
+# ------------------------------------------------------
+# --- Android CMake
+
+# download
+RUN mkdir /opt/android-cmake-tmp
+RUN cd /opt/android-cmake-tmp && wget -q https://dl.google.com/android/repository/cmake-3.6.3155560-linux-x86_64.zip -O android-cmake.zip
+# uncompress
+RUN cd /opt/android-cmake-tmp && unzip -q android-cmake.zip -d android-cmake
+# move to its final location
+RUN cd /opt/android-cmake-tmp && mv ./android-cmake ${ANDROID_HOME}/cmake
```

Comment:
Ok, I tested with this changes:
```
-RUN cd /opt/android-cmake-tmp && mv ./android-cmake ${ANDROID_HOME}/cmake
+RUN cd /opt/android-cmake-tmp && mkdir ${ANDROID_HOME}/cmake/ && mv ./android-cmake ${ANDROID_HOME}/cmake/3.6.3155560
---
-ENV PATH ${PATH}:${ANDROID_HOME}/cmake/bin
+ENV PATH ${PATH}:${ANDROID_HOME}/cmake/3.6.3155560/bin
```
It finds the binaries:
```
# which cmake
/opt/android-sdk-linux/cmake/3.6.3155560/bin/cmake
```
But it doesn't compile. It gives the same error as before:
```
* What went wrong:
A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_qaDebugApk'.
   > Failed to find CMake.
     Install from Android Studio under File/Settings/Appearance & Behavior/System Settings/Android SDK/SDK Tools/CMake.
     Expected CMake executable at /opt/android-sdk-linux/cmake/bin/cmake.
```